charts/backend/values.yaml

```
@@ -49,6 +49,17 @@ ingress:
       hosts:
         - api.signals.local
 
+probes:
+  backend:
+    livenessProbes:
```

Above I see `livenessProbe` instead of `livenessProbes`.


```suggestion
    livenessProbe:
```